### ISSUE TITLE
HTTPS refused to connect and MySQL can't be used

### ISSUE THREAD

**Issue by jasonong2019** [ONLYOFFICE/onlyoffice-owncloud#250]
Ubuntu 18.04 (fresh, only OpenSSH)

Next, follow blindly the tutorial. And docker mysql, createdb cloud

Use SQLite instead of MySQL. This happens when I tried to use MySQL

failed to connect to the database: an exception occured in driver: sqlstate [hy000] [2002] no such file or directory

End of the result. HTTP responds (on IE only. Google Chrome redirect it to HTTPS but since it does not work, it refused to connect), HTTPS refused to connect (OnlyOffice works)

In short,
I am frustated, this is as far as I can to install ownCloud w/ ONLYOFFICE on the same machine. Manual installation does not work (at first it worked, install the ownCloud, MySQL w/ no problem, install ONLYOFFICE Document Server, ownCloud broke, I am confused).

I can not edit /var/www/owncloud/config/config.php and et cetera (it does not exist). The tutorial become useless.

EDIT
This is supposed to be docker-owncloud-onlyoffice issue
